stories/test_data/chart_with_2_attributes_and_null_datapoints_request.json

```diff
@@ -0,0 +1,23 @@
+{
+    "execution": {
+        "afm": {
+            "measures": [{
+                "localIdentifier": "ah1EuQxwaCqs",
+                "definition": {"measure": {"item": {"identifier": "ah1EuQxwaCqs"}}}
+            }],
+            "attributes": [{
+                "displayForm": {"identifier": "label.product.id.name"},
+                "localIdentifier": "va_13"
+            }, {"displayForm": {"identifier": "label.stage.name.stagename"}, "localIdentifier": "va_17"}],
```

Comment:
Please fix the format. One attribute is properly formatted, the other is not. Whole JSON file format seems to be inconsistent when I'm looking at it.

Reply:
it's done :)

Reply:
It looks way better now. Please squash the commits and you can merge the pull request.